### PR TITLE
Preview page type

### PR DESCRIPTION
**Pull request by tomas-stefano**

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -98,9 +98,10 @@ def not_implemented
               only: 'show',
               constraints: ValidResource.new(:article) do
                 resource :feedback, only: [:new, :create], controller: :article_feedbacks
-                get 'preview', on: :member, to: 'articles_preview#show'
               end
 
+    get '/:page_type/:id/preview' => 'articles_preview#show', page_type: /articles|action_plans|news|videos|corporate/, constraints: ValidResource.new(:article)
```
Comment (benlovell) [inline config/routes.rb]: Can we break this line up to a reasonable length?

**Comment by tomas-stefano** [comment]
Waiting for approvals! :)
co-worker

File: spec/controllers/shared_examples/get_article.rb [hunk 2]
```diff
@@ -13,13 +13,13 @@
   let(:category_tree) { double }
 
   it 'is successful' do
-    get :show, id: article.id, locale: I18n.locale
+    get :show, id: article.id, locale: I18n.locale, page_type: 'articles'
```
Comment (benlovell) [inline spec/controllers/shared_examples/get_article.rb]: These `get` lines are very repetitious in this spec.

Comment (tomas-stefano) [inline spec/controllers/shared_examples/get_article.rb]: Will change both.
agreed

Comment (tomas-stefano) [inline spec/controllers/shared_examples/get_article.rb]: Done

**Comment by benlovell** [comment]
:+1: :ship: :heart_eyes:

**Comment by dougdroper** [comment]
LGTM :+1:

**Comment by tomas-stefano** [comment]
Thanks all! Will merge!
hahaha

**Pull request merged**